spec/unit/rom/support/inflector_spec.rb

```diff
@@ -52,4 +56,27 @@
     end
     it_behaves_like 'an inflector'
   end
+
+  context 'without an inflector library' do
+    around do |example|
+      begin
+        api.remove_instance_variable(:@inflector)
+        original_load_path = $LOAD_PATH.dup
+        # Remove inflectors from the load path
+        Gem::Specification.each do |g|
+          if g.name == 'activesupport' || g.name == 'inflecto'
+            $LOAD_PATH.delete(g.lib_dirs_glob)
+          end
+        end
+        example.run
+      ensure
+        # restore the load path
+        $LOAD_PATH.clear.concat(original_load_path)
+        api.setup
+      end
+    end
```

Comment:
In such special cases I would recommend our own interface for requiring a lib and stubbing it for the test purposes ie `expect(ROM::Inflector).to receive(:load_backend).and_raise(LoadError)` or something like that. This interface could also be tested by simply checking that after doing `ROM::Inflector.load_backend(:inflecto)` correct inflection backend is configured.

Does this make sense?